packages/types/src/policies.ts

```diff
@@ -0,0 +1,9 @@
+export interface Policy {
+	id: string;
+	name: string;
+	description: string;
+	enforce_tfa: null | boolean;
```

Comment:
Do we want to include an `icon`, similar to what `roles` currently have? Make it look nice and pretty 😊

Reply:
I wonder a bit whether it will be easy to find suitable/descriptive icons for policies 🤔 I'd probably leave it out of this PR, and then add it via the App PR if we see that it would actually be an enrichment.

Reply:
Sure!